### ISSUE TITLE
Question: How to set number of replicas and shards using thinky

### ISSUE THREAD

**Issue by jsfr** [neumino/thinky#571]
When running in production I would ideally like thinky to automatically set the number of replicas on created tables to 3. I however can't figure out how to do that. I have tried setting the options parameter of `createModel` to something like `{ replicas: 3 }` but the creation then just fails (even though there are three servers present and I am able to manually up the replicas through the rethinkdb admin module)

So basically: How do you set the number of replicas with `createModel`?

**Comment by neumino** [comment]
Can you pass the field `table` in the `options` of createModel
http://thinky.io/documentation/api/thinky/#createmodel

The options are passed in `r.tableCreate`
